qa/smoke-test-http/src/test/java/org/elasticsearch/http/snapshots/RestGetSnapshotsIT.java

```diff
@@ -208,10 +216,10 @@ private static Request baseGetSnapshotsRequest(String repoName) {
         return new Request(HttpGet.METHOD_NAME, "/_snapshot/" + repoName + "/*");
     }
 
-    private static List<SnapshotInfo> sortedWithLimit(String repoName,
-                                                      GetSnapshotsRequest.SortBy sortBy,
-                                                      int size,
-                                                      SortOrder order) throws IOException {
+    private static Tuple<String, List<SnapshotInfo>> sortedWithLimit(String repoName,
+                                                                     GetSnapshotsRequest.SortBy sortBy,
+                                                                     int size,
+                                                                     SortOrder order) throws IOException {
         final Request request = baseGetSnapshotsRequest(repoName);
```

Comment:
Can we replace this with `sortedWithLimit(repoName, sortBy, null, size, order);`?

Reply:
++